doc/content/integrations/webhooks/webhook-templates/adding_webhook_templates.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Adding Webhook Templates"
+description: ""
+weight: 4
+---
+
+{{% tts %}} uses webhook templates from the [`lorawan-webhook-templates` Github repository](https://github.com/TheThingsNetwork/lorawan-webhook-templates/).
+
+Once you have created a new webhook template with a proper [format]({{< ref "/integrations/webhooks/webhook-templates/format.md" >}}), you can easily test it locally by following next steps:
+
+1. Clone the [`lorawan-webhook-templates` Github repository](https://github.com/TheThingsNetwork/lorawan-webhook-templates/) to a local folder.
+
+2. Store your webhook template in previously mentioned folder.
+
+3. Include your webhook template in the `templates.yml` file.
+
+4. Update your {{% tts %}} configuration by adding following lines:
```

Comment:
I don't get this. What configuration? Is this a file? Where can people this file?

Reply:
As part of `DEVELOPMENT.md`, we do mention that people that run the stack, either using `go run` or through `docker`, may use the following config files:

https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.9/config/stack/ttn-lw-stack.yml

Reply:
@adriansmares Should we ignore the fact that people can run the stack without specifying the config file (like I was) and that in that case they should create the `.ttn-lw-stack.yml` file? Is this something we aren't supposed to be recommending?

Reply:
We should assume that people that setup the stack follow the instructions in `DEVELOPMENT.md`, which to refer to the custom configuration explicitly:
```
To customize the configuration, copy the configuration file /config/stack/ttn-lw-stack.yml to a different location (ex: the .env folder in your repo). The configuration is documented in the Configuration Reference.
```

Reply:
@laurensslats do you think this isn't clear enough?

Reply:
@nejraselimovic I would suggest you add a small reference, for example by adding a link to: /getting-started/installation/configuration/